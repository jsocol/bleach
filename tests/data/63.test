<image name=body><img src=x><svg onload=alert(1); autofocus>, <keygen onfocus=alert(1); autofocus>
