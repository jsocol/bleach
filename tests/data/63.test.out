<img><img src="x"><svg>, </svg>
